lib/extensions/ar_recent_records.rb

```diff
@@ -0,0 +1,26 @@
+module ArRecentRecords
+  extend ActiveSupport::Concern
+
+  module ClassMethods
```

Comment:
You might want to add a `created_timestamp_column` method to this as well, and reference that in the strings for the SQL.  Pretty sure we still have some `created_at` columns out there as well.

Not sure if it makes sense to define that as just a class variable, and we just do a lookup on class load or something, or just have classes that need to override it do so (leaning towards the second one to be honest).